_includes/_code/grid/main.dart

```diff
@@ -0,0 +1,104 @@
+import 'package:flutter/material.dart';
+//import 'package:flutter/rendering.dart' show debugPaintSizeEnabled;
+
+void main() {
+  //debugPaintSizeEnabled = true;
+  runApp(new MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return new MaterialApp(
+      title: 'Flutter Demo',
+      theme: new ThemeData(
+        primarySwatch: Colors.blue,
+      ),
+      home: new MyHomePage(title: 'Flutter Demo Home Page'),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
+  MyHomePage({Key key, this.title}) : super(key: key);
+
+  final String title;
+
+  @override
+  _MyHomePageState createState() => new _MyHomePageState();
+}
+
+// The images are saved with names pic1.jpg, pic2.jpg...pic28.jpg.
+// The List.generate constructor allows an easy way to create
+// a list when objects have a predictable naming pattern.
+List<Container> _buildGridTileList(int count) {
+  List<Container> containers;
+
+  containers = new List<Container>.generate(
+      count,
+      (container) =>
+          new Container(child: new Image.asset('images/pic${count--}.jpg')));
```

Comment:
Also, no need for the `containers` variable.  You can just return the result of `generate` directly as the result of the function.